docs/api/navigators/StackNavigator.md

```diff
@@ -108,6 +108,10 @@ String or React Element used by the header. Defaults to scene `title`
 
 Title string used by the back button on iOS, or `null` to disable label. Defaults to the previous scene's `headerTitle`
 
+#### `headerBackImage`
+
+The image source (either a remote URL {{uri: 'path/to/image'}} or a local file resource  {require('path/to/image')}) used by the back button.
```

Comment:
Please back backticks to code part
Example:
{{uri: 'path/to/image'}} vs `{{uri: 'path/to/image'}}`